homeassistant/components/synology/camera.py

```diff
@@ -62,7 +62,7 @@
     # add cameras
     devices = []
     for camera in cameras:
-        if not config.get(CONF_WHITELIST):
+        if not config.get(CONF_WHITELIST) or camera.name in config.get(CONF_WHITELIST):
```

Comment:
Use `dict[key]` the second time we access the dict, since we know the key is in the dict by then.

Reply:
Isn't this the case for both (not just the second)? Since voluptuous takes care of that already...

Reply:
```suggestion
        if not config[CONF_WHITELIST] or camera.name in config[CONF_WHITELIST]:
```

I adapted the PR underneath already - just trying out this suggest feature.